CapitalForecasting/Modules/processing_and_eda.R

```diff
@@ -0,0 +1,103 @@
+library(tidyverse)
+library(tsibble)
+library(fable)
+library(GGally)
+library(feasts)
+
+
+# making some fake data
+set.seed(42)
+date <- seq(as.Date("2010-01-01"), as.Date("2020-01-01"), "weeks")
+# x <- rnorm(n = length(date), mean = 10, sd = 5)
+# y <- x * 2 + 10 * rnorm(length(date))
+time_series_data <- function(mean, sd) {
+  # should make this more seasonal
+  column <- c(0)
+  for (i in 1:length(date)) {
+    if (i %% 7 == 0) {
+      column <- c(column, column[i - 1] + rnorm(1, 7, 2))
+    } else{
+      column <- c(column, column[i - 1] + rnorm(1, mean, sd))
+    }
+  }
+  return(column)
+}
+current_assets <- time_series_data(2, 30)
+current_liabilities <- time_series_data(1, 20)
+
```

Comment:
Interesting data simulation. Do you mind providing some resources you used to design the structure of `time_series_data()`? Would this be similar to a Monte Carlo simulation?

Reply:
I kind of just threw it together so I could start graphing with a tiny bit of seasonality. It's not super useful, I just wanted to  see what my graphs look like.

Reply:
Sounds good, just keep in mind that if you are planning on demoing the output that the numbers should more or less make sense. In other words, totally fine if they are random so long as they are believable.